### PR TITLE
Gagan work

### PR DESCRIPTION
**Pull request by GaganDureja**
This PR have multiple commits as following:

- User app created.
- Abstractuser model added to make changes in default user model.
- User appname added in url file and table migration done.
- Tweet app created and models added.
- home redirect and login authentication check view added with default empty page

### CODE REVIEW

**Comment by grsahil20** [comment]
PR title should say what is happening in the PR

File: user/migrations/0001_initial.py
Comment (grsahil20) [inline user/migrations/0001_initial.py:1]: Can we break this into different migrations and name them what are they diung

File: user/models.py
Comment (grsahil20) [inline user/models.py:1]: Use extension in VSCODE https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces

automatically removes whitespaces

**Comment by grsahil20** [comment]
Rest review after you have changed the branch name and updated the PR accordingly

**Comment by GaganDureja** [comment]
> PR and Branch name should say what is happening in the PR

Okay

File: user/migrations/0001_initial.py
Comment (GaganDureja) [reply user/migrations/0001_initial.py:1]: > Can we break this into different migrations and name them what are they doing

migration is done for user as default user is already there in base django project so to override one migration is done and will refer in future with some name or description

File: user/models.py
Comment (GaganDureja) [reply user/models.py:1]: done

**Pull request merged**